tests/core/pyspec/eth2spec/test/helpers/voluntary_exits.py

```diff
@@ -1,10 +1,14 @@
 from random import Random
 from eth2spec.utils import bls
+from eth2spec.test.context import expect_assertion_error
 from eth2spec.test.helpers.keys import privkeys
 
 
-def prepare_signed_exits(spec, state, indices):
-    domain = spec.get_domain(state, spec.DOMAIN_VOLUNTARY_EXIT)
+def prepare_signed_exits(spec, state, indices, fork_version=None):
```

Comment:
Where is this  being used with `fork_version` not equal to `None`?
Did you upddate the helper but not end up needing the altered functionality?

Reply:
Or is it just to mirror `sign_voluntary_exit` regardless of use?

Reply:
just a mirror here